clang/test/SemaCXX/GH63151.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clang_cc1 -fsyntax-only -verify %s
-
+// RUN: %clang_cc1 -fsyntax-only -Wno-c++11-narrowing-const-reference -verify=allow %s
```

Comment:
"allow" is slightly confusing as the verify prefix. Maybe use `-verify=narrowing` and `-verify=narrowing-constref` for these two clang invocations? In any case, this is just minor cosmetic stuff.